packages/p2p/src/components/buy-sell/__tests__/buy-sell-header.spec.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { fireEvent, screen, render } from '@testing-library/react';
+import { useStores } from 'Stores';
+import BuySellHeader from '../buy-sell-header.jsx';
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn(),
+}));
+
+const mockProps = () => ({
```

Comment:
We don't follow the same naming convention (`snake_case`) for tests?

Reply:
@salarhfz-fs In case of functions i believe we use camelCase
@artsemkavaleu-deriv maybe it will be better to use:
const props = { your props } instead of invoke mockProps function in every test case,

Reply:
sure, done, thx for notice!